### ISSUE TITLE
Qt 6.5: issues with dev tools

### ISSUE THREAD

**Issue by jpalus** [qutebrowser/qutebrowser#7676]
**Version info**:
```
qutebrowser v2.5.4
Git commit: 
Backend: QtWebEngine 6.5, based on Chromium 108.0.5359.220 (from api)
Qt: 6.5.0
```

**Does the bug happen if you start with `--temp-basedir`?**: Yes

**Description**
It appears that right click on various elements does not invoke proper context menu, ie I would expect an option to clear cookies when right clicking in cookies section but it presents standard web page right click menu:
![dev-tools-right-click](https://user-images.githubusercontent.com/484155/234545675-fc87f5f8-12fd-4917-ada7-815ca77e2ad9.jpg)

Another minor issue -- clicking closing cross in dev tools does nothing. dev tools are not hidden.
![screenshotwl-20230426121720](https://user-images.githubusercontent.com/484155/234546039-1b4aca4d-70c6-4d1b-82fa-1e33a6b76124.png)

**Comment by The-Compiler** [comment]
Confirmed. Seems like an upstream bug, don't think there is much qutebrowser can do about it.

What's both more funny and more annoying is that **left**-clicking the » to expand more tabs *also* opens the context menu instead:

![image](https://user-images.githubusercontent.com/625793/234601075-4b2c3266-decc-444f-a1f5-d03584d2b86c.png)

**Comment by The-Compiler** [comment]
Reported upstream: [[QTBUG-113251] [REG 6.4 -> 6.5] Context menu functionality in devtools broken - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-113251)

Don't think there is anything qutebrowser can do to work around this.

**Comment by The-Compiler** [comment]
Sadly the upstream fix got lost and didn't make it into 6.5.2, but it was [merged now](https://codereview.qt-project.org/c/qt/qtwebengine/+/487008) and should be in 6.5.3, [planned for](https://wiki.qt.io/Qt_6.5_Release) early September.

**Comment by The-Compiler** [comment]
True. That might need to be reported upstream separately...

**Comment by jpalus** [comment]
Created QTBUG-115976.

**Comment by The-Compiler** [comment]
Thanks! Here is a link: [[QTBUG-115976] Dev tools not hidden with closing cross - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-115976)

**Comment by The-Compiler** [comment]
Looks like there is a fix incoming for Qt 6.6: [Fix close button in devtools (If58a96e3) · Gerrit Code Review](https://codereview.qt-project.org/c/qt/qtwebengine/+/495739)

**Comment by jpalus** [comment]
Thanks @The-Compiler. Confirmed the fix works fine.
